app/src/components/Sidebar.svelte

```diff
@@ -32,10 +35,16 @@
 
   :global(li .show-on-hover) {
     opacity: 0;
+    display: none;
+  }
+
+  :global(li:hover svg) {
```

Comment:
Doesn't `:global` makeit apply to any li in the entire app?

Reply:
It applies to any svg that lives under a `li:hover`. This is what I was talking about regarding svelte not supporting HOC or style overrides: https://github.com/sveltejs/svelte/issues/2870.

So far the only way to prevent svelte from GCing styles is to make them global.
Let me know if you know of a better workaround.

Reply:
In this specific case can't we move the `li` to `SidebarItem` and move the style with it?

Reply:
Not really, because `SidebarAvatar` also needs the same functionality. What I could do is move everything back into sidebar, because `SidebarAvatar` and `SidebarItem` don't really make sense as separate components. Should I do that?

Reply:
Another way to think about it is that this hover behaviour should be consistent in the app, unless overriden locally bei a component. So it might be worth having it in the global styles.

Reply:
I think I'll leave it as it is for now to not lose more time on it. We can re-think it if we run into problems or if there is a new svelte release which addresses this functionality.